### PR TITLE
add new COALTON-LIBRARY/SYSTEM package and gc, time, sleep functions

### PR DESCRIPTION
**Pull request by stylewarning**

### CODE REVIEW

**Review by eliaslfox — APPROVED** [review]
This looks good. Personally I don't think these are worth importing in the prelude.

**Comment by stylewarning** [comment]
> This looks good. Personally I don't think these are worth importing in the prelude. Especially gc and sleep.

True, I agree, I'll remove them.

**Comment by stylewarning** [comment]
I kept TIME because I find it so useful in day-to-day development, but duly removed GC and SLEEP.

File: library/prelude.lisp [hunk 1]
```diff
@@ -156,7 +156,17 @@
...
+   #:Iterator)
+  (:export
+   #:Iterator)
+
+  (:import-from
+   #:coalton-library/system
+   #:time)
+  (:export
+   #:gc
+   #:time
```
Comment (eliaslfox) [inline library/prelude.lisp:168]: I think `gc` and `sleep` shouldn't be exported here

**Pull request merged**